packages/moonstone/styles/mixins.less

```diff
@@ -288,22 +288,8 @@
 	});
 }
 
-.moon-popup-header-text() {
```

Comment:
We need to document that this was removed.

Reply:
This may be my fault. I had some difficulty resolving the merge conflict and I may have unintentionally dropped that line.